<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralPageWidget</class>
 <widget class="QWidget" name="GeneralPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Hardware Overview</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="profileRotationBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Profile rotation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="lwRotationList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QPushButton" name="pbDown">
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pbRemoveFromRotationList">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pbAddToRotationList">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pbUp">
            <property name="text">
             <string>Up</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shortcutGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Shortcuts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbAddToRotationList</sender>
   <signal>clicked()</signal>
   <receiver>GeneralPageWidget</receiver>
   <slot>profileAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDown</sender>
   <signal>clicked()</signal>
   <receiver>GeneralPageWidget</receiver>
   <slot>profileDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRemoveFromRotationList</sender>
   <signal>clicked()</signal>
   <receiver>GeneralPageWidget</receiver>
   <slot>profileRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbUp</sender>
   <signal>clicked()</signal>
   <receiver>GeneralPageWidget</receiver>
   <slot>profileUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>profileChanged()</slot>
  <slot>profileUp()</slot>
  <slot>profileDown()</slot>
  <slot>profileRemove()</slot>
  <slot>profileAdd()</slot>
 </slots>
</ui>
